2021 prague speakers (#1562)
writethedocs/www: docs/conf/prague/2021/news/announcing-speakers.rst

:template: {{year}}/generic.html

> .. post:: July 28, 2021

Author: ```suggestion
.. post:: July 30, 2021
```

   :tags: prague-2021, speakers, tickets, sponsors

Announcing speakers
===================

Hello, documentarians! Today we're happy to announce the full list of talks and speakers for the 2021 Prague virtual conference. 

Every year talk selection gets more and more difficult as we get a larger, stronger batch of submissions.
Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you!

Full speaker line-up
--------------------

> Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.

Author: I'm not super keen on "documentarian", or even "role of documentation". How about 

> Because being a documentarian can mean different things to different people, we aim for a line-up that

Reviewer: > I'm not super keen on "documentarian", or even "role of documentation". How about
> 
> > Because being a documentarian can mean different things to different people, we aim for a line-up that

Yes, fully agree. I can live with the current text too though.

Reviewer 2: ```suggestion
Every year, we look to bring a wide range of voices to the Write the Docs stage. Because being a documentarian can mean different things to different people, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
```
